syft/execution/plan.py

```diff
@@ -137,6 +137,10 @@ def __init__(
         if not hasattr(self, "forward"):
             self.forward = forward_func or None
 
+        self.wrapped_framework = {}
+        for f_name, f_packages in framework_packages.items():
+            self.wrapped_framework[f_name] = FrameworkWrapper(f_packages, self.role, self.owner)
+
```

Comment:
Maybe a line of comment would be helpful to better understand the purpose of this

Reply:
Done